src/main/java/org/eclipse/yasson/internal/model/customization/StrategiesProvider.java

```diff
@@ -0,0 +1,100 @@
+package org.eclipse.yasson.internal.model.customization;
+
+import static java.util.stream.Collectors.toList;
+import static java.util.Comparator.comparing;
+import static javax.json.bind.config.PropertyNamingStrategy.*;
+import static javax.json.bind.config.PropertyOrderStrategy.*;
+
+import javax.json.bind.JsonbException;
+import javax.json.bind.config.PropertyNamingStrategy;
+import org.eclipse.yasson.internal.model.PropertyModel;
+import org.eclipse.yasson.internal.properties.Messages;
+import org.eclipse.yasson.internal.properties.MessageKeys;
+
+import java.nio.CharBuffer;
+import java.util.function.Function;
+import java.util.*;
+
+public final class StrategiesProvider {
+    private StrategiesProvider() {}
+    
+    public static final PropertyNamingStrategy CASE_INSENSITIVE_STRATEGY = Objects::requireNonNull;
+    
+    public static Function<Collection<PropertyModel>, List<PropertyModel>> getOrderingFunction(String strategy){
+        switch(strategy) {

```

Comment:
This might be personal preference, but I think that having those return statements on the new line is much cleaner. Don't you think?  :-)

Reply:
I saw in some other files that did the same thing (putting the return on a new line), so I can change it to that.